### PR TITLE
Dashboard Terraform

### PR DESCRIPTION
**Pull request by emily-curtis**
In this pull request:

- The `dashboard.tf` file was created, which creates the necessary resources needed on AWS to run the dashboard.
- `provider.tf` and `variables.tf` were moved into the 'dashboard' folder (can be removed if needed).

### CODE REVIEW

File: terraform/dashboard/dashboard.tf [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# Variables for VPC and Subnets
+variable "vpc_id" {
+  description = "The ID of the existing VPC"
+}
+
+variable "subnet_ids" {
+  description = "The IDs of the existing subnets"
+  type        = list(string)
+}
```
Comment (ebradley12) [inline terraform/dashboard/dashboard.tf:9]: Could these variables go into the variables file rather than the main script?

Comment (emily-curtis) [reply terraform/dashboard/dashboard.tf:9]: It is in the variables file, it's just calling the names of them from `variables.tf` into the main script!

File: terraform/dashboard/dashboard.tf [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+# Create an ECR repository
+resource "aws_ecr_repository" "c14_bandcamp_dashboard_repo" {
+  name = "c14-bandcamp-dashboard-repo"
+}
```
Comment (ebradley12) [inline terraform/dashboard/dashboard.tf:14]: Repo already created in a separate script called 'c14-bandcamp-dashboard-ecr'

**Review by ebradley12 — APPROVED** [review]
This looks great!!

File: terraform/dashboard/dashboard.tf (hunk 1 above)
Comment (bentrzcinski) [reply terraform/dashboard/dashboard.tf:9]: I don't think you need to do that. If they are in the variables.tf script you should be able to reference them using 'var.xxx' without importing them like this.

**Review by bentrzcinski — COMMENTED** [review]
otherwise looks great

**Pull request merged**